js/test/Common/ControllerI.js

```diff
@@ -114,12 +113,12 @@ class ProcessControllerI extends Test.Common.BrowserProcessController {
     }
 
     redirect(url, current) {
-        current.con.close(Ice.ConnectionClose.Gracefully);
+        current.con.close();
```

Comment:
We forget to fix this when we remove ConnectionClose enum, browser tests has been broken for a while.

Would be worth to investigate if we can run them in CI.